util/chunk/chunk_test.go

```diff
@@ -32,6 +32,7 @@ import (
 	"github.com/pingcap/tidb/sessionctx/stmtctx"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/types/json"
+	"github.com/stretchr/testify/require"
 )
 
 func TestT(t *testing.T) {
```

Comment:
Since we write this part of logic in `TestMain` already, we can keep only `check.TestingT(t)` in `TestT`